2022-04-25 10:36:03,631 INFO    MainThread:131956 [wandb_setup.py:_flush():75] Loading settings from /home/leeandy/.config/wandb/settings
2022-04-25 10:36:03,632 INFO    MainThread:131956 [wandb_setup.py:_flush():75] Loading settings from /home/leeandy/Deep_learning_lab/lab5/wandb/settings
2022-04-25 10:36:03,632 INFO    MainThread:131956 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-25 10:36:03,632 INFO    MainThread:131956 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'lab5/main.py', 'program': 'main.py'}
2022-04-25 10:36:03,632 INFO    MainThread:131956 [wandb_init.py:_log_setup():438] Logging user logs to /home/leeandy/Deep_learning_lab/lab5/wandb/run-20220425_103603-2jx88w0p/logs/debug.log
2022-04-25 10:36:03,632 INFO    MainThread:131956 [wandb_init.py:_log_setup():439] Logging internal logs to /home/leeandy/Deep_learning_lab/lab5/wandb/run-20220425_103603-2jx88w0p/logs/debug-internal.log
2022-04-25 10:36:03,633 INFO    MainThread:131956 [wandb_init.py:init():472] calling init triggers
2022-04-25 10:36:03,633 INFO    MainThread:131956 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-04-25 10:36:03,633 INFO    MainThread:131956 [wandb_init.py:init():525] starting backend
2022-04-25 10:36:03,633 INFO    MainThread:131956 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-25 10:36:03,650 INFO    MainThread:131956 [backend.py:ensure_launched():219] starting backend process...
2022-04-25 10:36:03,655 INFO    MainThread:131956 [backend.py:ensure_launched():224] started backend process with pid: 131979
2022-04-25 10:36:03,656 INFO    MainThread:131956 [wandb_init.py:init():534] backend started and connected
2022-04-25 10:36:03,659 INFO    MainThread:131956 [wandb_init.py:init():598] updated telemetry
2022-04-25 10:36:03,670 INFO    MainThread:131956 [wandb_init.py:init():629] communicating run to backend with 30 second timeout
2022-04-25 10:36:04,345 INFO    MainThread:131956 [wandb_run.py:_on_init():1922] communicating current version
2022-04-25 10:36:04,506 INFO    MainThread:131956 [wandb_run.py:_on_init():1926] got version response upgrade_message: "wandb version 0.12.15 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-25 10:36:04,506 INFO    MainThread:131956 [wandb_init.py:init():660] starting run threads in backend
2022-04-25 10:36:04,541 INFO    MainThread:131956 [wandb_run.py:_console_start():1896] atexit reg
2022-04-25 10:36:04,542 INFO    MainThread:131956 [wandb_run.py:_redirect():1769] redirect: SettingsConsole.REDIRECT
2022-04-25 10:36:04,542 INFO    MainThread:131956 [wandb_run.py:_redirect():1774] Redirecting console.
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_run.py:_redirect():1830] Redirects installed.
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_init.py:init():685] run started, returning control to user process
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_config.py:__setitem__():153] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f994f775f10>>
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_run.py:_config_callback():1129] config_cb learning_rate 0.001 None
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_config.py:__setitem__():153] config set batch_size = 12 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f994f775f10>>
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_run.py:_config_callback():1129] config_cb batch_size 12 None
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_config.py:__setitem__():153] config set epochs = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f994f775f10>>
2022-04-25 10:36:04,543 INFO    MainThread:131956 [wandb_run.py:_config_callback():1129] config_cb epochs 10 None
2022-04-25 10:36:06,488 INFO    MainThread:131956 [wandb_watch.py:watch():47] Watching
2022-04-25 10:36:14,064 INFO    MainThread:131956 [wandb_run.py:_atexit_cleanup():1865] got exitcode: 255
2022-04-25 10:36:14,065 INFO    MainThread:131956 [wandb_run.py:_restore():1837] restore
